app-react/src/main/kotlin/com/ustadmobile/view/SiteDetailScreen.kt

```diff
@@ -0,0 +1,90 @@
+package com.ustadmobile.view
+
+import com.ustadmobile.core.generated.locale.MessageID
+import com.ustadmobile.core.hooks.useStringsXml
+import com.ustadmobile.core.viewmodel.PersonDetailUiState
+import com.ustadmobile.core.viewmodel.SiteDetailUiState
+import com.ustadmobile.lib.db.entities.*
+import com.ustadmobile.mui.components.UstadDetailField
+import csstype.px
+import mui.icons.material.*
+import mui.material.*
+import mui.material.styles.TypographyVariant
+import mui.system.Stack
+import mui.system.StackDirection
+import mui.system.responsive
+import react.FC
+import react.Props
+import react.create
+import react.dom.html.ReactHTML
+
+external interface SiteDetailProps : Props {
+    var uiState: SiteDetailUiState
+    var onClickLang: (SiteTermsWithLanguage) -> Unit
+}
+
+val SiteDetailComponent2 = FC<SiteDetailProps> { props ->
+
+    val strings = useStringsXml()
+
+    Container {
+        maxWidth = "lg"
+
+        Stack {
+            direction = responsive(StackDirection.column)
+            spacing = responsive(10.px)
+
+            UstadDetailField {
+                icon = AccountBalanceRounded.create()
+                labelText = strings[MessageID.name]
+                valueText = props.uiState.site?.siteName.toString()
+            }
+
+            UstadDetailField {
+                icon = ManageSearchOutlined.create()
+                labelText = "Guest login enabled"
+                valueText = props.uiState.site?.guestLogin.toString()
+            }
+
+            UstadDetailField {
+                icon = HowToRegRounded.create()
+                labelText = "Registration allowed"
+                valueText = props.uiState.site?.registrationAllowed.toString()
+            }
+
+            Typography {
+                variant = TypographyVariant.h6
+                + strings[MessageID.terms_and_policies]
+            }
+
+            props.uiState.siteTerms.forEach { item ->
+                ListItem {
+                    var item = item
```

Comment:
No need for var item = item, because it is already named as part of the forEach line.